openc3-cosmos-init/plugins/packages/openc3-vue-common/src/tools/base/Notifications.vue

```diff
@@ -51,13 +51,15 @@
             icon="mdi-notification-clear-all "
             variant="text"
             data-test="clear-notifications"
+            aria-label="Clear All Notifications"
             @click="clearNotifications"
```

Comment:
I know we just cleaned up all the unnecessary notifications but this triple line icon I think could use a tooltip ... it's not immediately obvious to me what it does and when you click it all the notifications go poof.